app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderPresenter.kt

```diff
@@ -568,13 +568,16 @@ class ReaderPresenter(
 
         // Build destination file.
         val filenameSuffix = " - ${page.number}.${type.extension}"
-        val filenamePrefix = if (preferences.addSpoiler()) "SPOILER_" else ""
         val validFilename = DiskUtil.buildValidFilename(
-            "${manga.title} - ${chapter.name}".takeBytes(MAX_FILE_NAME_BYTES - filenameSuffix.byteSize())
+            prefix + (
+                "${manga.title} - " +
+                    "${chapter.name}"
+                ).takeBytes(
+                MAX_FILE_NAME_BYTES - filenameSuffix.byteSize()
+            ) + filenameSuffix
```

Comment:
Why you split the string to concatenate instead of just using a string template?
```suggestion
            prefix + 
                    ("${manga.title} - ${chapter.name}")
                        .takeBytes(MAX_FILE_NAME_BYTES - filenameSuffix.byteSize()) + 
                    filenameSuffix
```